<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectBorder</class>
 <widget class="QWidget" name="selectBorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="VideoLabel" name="videoLabel">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="text">
      <string>videoLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioTL">
       <property name="text">
        <string>Top Left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioTR">
       <property name="text">
        <string>Top Right</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioBL">
       <property name="text">
        <string>Bottom Left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioBR">
       <property name="text">
        <string>Bottom Right</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Field</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Goal</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Depth:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Border:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="width">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="height">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="goalDepth"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="border">
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="goalSize">
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="proportion">
         <property name="singleStep">
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Proportion:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoLabel</class>
   <extends>QLabel</extends>
   <header location="global">videolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
